fatiando/gravmag/transform.py

```diff
@@ -591,7 +591,8 @@ def radial_average_spectrum(kx, ky, pds, max_radius=None, ring_width=None):
     if max_radius is None:
         max_radius = min(kx.max(), ky.max())
     if ring_width is None:
-        ring_width = max(kx[1, 0], ky[0, 1])
+        ring_width = max(numpy.unique(kx)[numpy.unique(kx) > 0][0],
+                         numpy.unique(ky)[numpy.unique(ky) > 0][0])
```

Comment:
@santis19 thanks for the fix! I just noticed that the docstring is wrong. It describes `data` and `shape` when the function receives `pds`. Would you mind fixing this as well? Let me know and I'll merge the PR.

Reply:
@leouieda, thanks for noticing it, it seems that I was more aware of the code than the documentation.
Now it's fixed. If you like how the new documentation is written, merge it!